virt/virtual_machines/virtual_disks/virt-managing-offline-vm-snapshots.adoc

```diff
@@ -0,0 +1,30 @@
+[id="virt-managing-offline-vm-snapshots"]
+= Managing offline virtual machine snapshots
+include::modules/virt-document-attributes.adoc[]
+:context: virt-managing-offline-vm-snapshots
+toc::[]
+
+You can create, restore, and delete virtual machine (VM) snapshots for VMs that are powered off (offline). {VirtProductName} supports
+offline VM snapshots on:
+
+* Red Hat OpenShift Container Storage
+* Any other storage provider with the Container Storage Interface (CSI) driver supporting the Kubernetes Volume Snapshot API
+
+include::modules/virt-about-vm-snapshots.adoc[leveloffset=+1]
+
+include::modules/virt-creating-offline-vm-snapshot-cli.adoc[leveloffset=+1]
+
+include::modules/virt-restoring-vm-from-snapshot-cli.adoc[leveloffset=+1]
+
+include::modules/virt-deleting-vm-snapshot-cli.adoc[leveloffset=+1]
+
+== Additional resources
+
+* xref:../../../storage/container_storage_interface/persistent-storage-csi-snapshots.adoc#persistent-storage-csi-snapshots[CSI Volume Snapshots]
+
+* The link:https://kubevirt.io/api-reference/v0.33.0/definitions.html#_v1alpha1_virtualmachinesnapshot[KubeVirt API Reference] provides broader context for the parameters and hierarchy of the VM snapshot CRDs.
+
+[NOTE]
+====
+The KubeVirt API Reference is the upstream project reference and might contain parameters that are not supported in {VirtProductName}.
```

Comment:
Just wondering whether we should be linking to this upstream doc if there are some differences and things that aren't supported. It might be best not to, but if you have any objections to removing it, we can always run it by our support contacts.

Reply:
I wasn't sure about this one. I think it is best to remove the link.